Bump to version 0.5.11 with working tests

pkgw opened the pull request:
I hope. I really think so.

pkgw committed:
Update to version 0.5.10 and, I hope, finally fix tests.

pkgw committed:
MNT: Re-rendered with conda-smithy 2.3.0

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

pkgw committed:
recipe/forge_test.sh: cut out tests to run faster

We're exceeding the 120-minute run time limitation on Circle between
the builds of the three Python versions.

pkgw committed:
Update to upstream version 0.5.11

pkgw committed:
recipe/forge_test.sh: cut out a couple more runs

Last attempt missed the time cutoff by about 10 minutes. Hopefully this will
get us under.

pkgw committed:
recipe/forge_test.sh: don't call test.py

It just takes too long.

pkgw committed:
recipe/meta.yaml: collect timing information for the test script

This should be helpful going forward.
